crates/runtime_isolate/Cargo.toml

```
@@ -15,10 +15,21 @@ log = { version = "0.4.18", features = ["std", "kv_unstable"] }
 once_cell = "1.17.1"
 async-recursion = "1.0.4"
 linked-hash-map = "0.5.6"
+flate2 = "1.0.26"
+serde_v8 = "0.100.0"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+uuid = { version = "1.3.3", features = ["v4"] }
```

Could you update this dep to match `runtime_crypto/Cargo.toml`?
```suggestion
uuid = { version = "1.3.4", features = ["v4", "fast-rng"] }
```

---

done